<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphic Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, 
    x2: 1, y2: 1, 
    stop: 0 #FF2E4C, 
    stop: 1 #1E2A78
);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
background: transparent;      /* Без фона */
    color: white;               /* Цвет текста */
    border: 3px solid white;    /* Белая рамка */
    border-radius: 10px;        /* Закругленные углы */
    padding: 11px;              /* Отступы */
	font-size: 20px; 
}</string>
    </property>
    <property name="title">
     <string>Типы функций</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-family: Cambria;
font-size: 20px;
background: transparent;</string>
     </property>
     <property name="text">
      <string>Многочлен</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-family: Cambria;
font-size: 20px;
background: transparent;</string>
     </property>
     <property name="text">
      <string>Тригонометрическая</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font-family: Cambria;
font-size: 20px;
</string>
     </property>
     <property name="text">
      <string>Экспотенциальная</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>211</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font-family: Cambria;
font-size: 20px;
</string>
     </property>
     <property name="text">
      <string>Логарифмическая</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font-family: Cambria;
font-size: 20px;
</string>
     </property>
     <property name="text">
      <string>Модуль</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
background: transparent;      /* Без фона */
    color: white;               /* Цвет текста */
    border: 3px solid white;    /* Белая рамка */
    border-radius: 10px;        /* Закругленные углы */
    padding: 11px;              /* Отступы */
	font-size: 20px; 
}</string>
    </property>
    <property name="title">
     <string>Диапазон</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;               /* Цвет текста */
border: 2px solid white;    /* Белая рамка */</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;               /* Цвет текста */
border: 2px solid white;    /* Белая рамка */</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font-family: Cambria;
font-size: 20px;
</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;
font-family: Cambria;
font-size: 20px;
</string>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: transparent;    /* Без фона */
    color: white;               /* Цвет текста */
    border: 3px solid white;    /* Белая рамка */
    border-radius: 25px;        /* Закругленные углы */
    padding: 11px;              /* Отступы */
	color: white;
	font-family: Cambria;
	font-size: 20px;
}

QPushButton:hover {
    background: transparent;    /* Без фона при наведении */
    border: 4px solid white;     /* Увеличенная белая рамка при наведении */
    text-decoration: underline;   /* Подчеркнутый текст при наведении */
}

</string>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: transparent;    /* Без фона */
    color: white;               /* Цвет текста */
    border: 3px solid white;    /* Белая рамка */
    border-radius: 25px;        /* Закругленные углы */
    padding: 11px;              /* Отступы */
	color: white;
	font-family: Cambria;
	font-size: 20px;
}

QPushButton:hover {
    background: transparent;    /* Без фона при наведении */
    border: 4px solid white;     /* Увеличенная белая рамка при наведении */
    text-decoration: underline;   /* Подчеркнутый текст при наведении */
}</string>
    </property>
    <property name="text">
     <string>Очистить поле</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>560</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: transparent;    /* Без фона */
    color: white;               /* Цвет текста */
    border: 3px solid white;    /* Белая рамка */
    border-radius: 25px;        /* Закругленные углы */
    padding: 11px;              /* Отступы */
	color: white;
	font-family: Cambria;
	font-size: 20px;
}

QPushButton:hover {
    background: transparent;    /* Без фона при наведении */
    border: 4px solid white;     /* Увеличенная белая рамка при наведении */
    text-decoration: underline;   /* Подчеркнутый текст при наведении */
}</string>
    </property>
    <property name="text">
     <string>Скачать график</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>631</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
background: transparent;      /* Без фона */
    color: white;               /* Цвет текста */
    border: 3px solid white;    /* Белая рамка */
    border-radius: 10px;        /* Закругленные углы */
    padding: 11px;              /* Отступы */
	font-size: 20px; 
}</string>
    </property>
    <property name="title">
     <string>График</string>
    </property>
    <widget class="GraphicWidget" name="graphicWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>611</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    border-radius: 10px; </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
background: transparent;      /* Без фона */
    color: white;               /* Цвет текста */
    border: 3px solid white;    /* Белая рамка */
    border-radius: 10px;        /* Закругленные углы */
    padding: 11px;              /* Отступы */
	font-size: 20px; 
}</string>
    </property>
    <property name="title">
     <string>Ввод функции:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: white;               /* Цвет текста */
border: 2px solid white;    /* Белая рамка */</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent;    /* Без фона */
    color: white;               /* Цвет текста */
    border: 2px solid white;    /* Белая рамка */
    padding: 3px;              /* Отступы */
	color: white;
	font-family: Cambria;
	font-size: 20px;
}

QPushButton:hover {
    background: transparent;    /* Без фона при наведении */
    border: 3px solid white;     /* Увеличенная белая рамка при наведении */
    font-weight: bold;   /* Подчеркнутый текст при наведении */
}


</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicWidget</class>
   <extends>QWidget</extends>
   <header location="global">graphicwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
